dg/media/configs.js

```diff
@@ -1140,10 +1140,12 @@ function() {
 
     var misc = {
         download_chunk_size: 2000,
+        data_transfer_schema: "uploadqueue",
+        data_transfer_credentials_schema: "user",
```

Comment:
But things that wont change in future/ should not be changed anyway, should be included in library and option to change should not be given.